node/src/chain_spec/neumann.rs

```diff
@@ -389,11 +376,8 @@ fn testnet_genesis(
 		},
 		balances: neumann_runtime::BalancesConfig { balances: endowed_accounts },
 		parachain_info: neumann_runtime::ParachainInfoConfig { parachain_id: para_id },
-		collator_selection: neumann_runtime::CollatorSelectionConfig {
-			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-			candidacy_bond: collator_bond,
-			..Default::default()
-		},
+		// Defaults to active collators from session pallet unless configured otherwise
+		parachain_staking: Default::default(),
```

Comment:
won't this only be used when standing up new NEU test chains now? Shouldn't we configure this to something that makes sense for that?

Reply:
We can definitely configure this to something that makes sense for that but I'm not sure what that is. Leaving it as `Default::default()` means that parachain staking is going to defer to the session pallet/ invulnerables. Regardless of whether this stays as default or not I'd prefer to make that change in a separate pr since it won't have any bearing on the runtime upgrade.

Reply:
Made this issue to track this work- https://oak-network.atlassian.net/browse/ENG-360